src/Text/Trifecta/Rope.hs

```diff
@@ -59,22 +78,69 @@ data Rope = Rope !Delta !(FingerTree Delta Strand) deriving Show
 rope :: FingerTree Delta Strand -> Rope
 rope r = Rope (measure r) r
 
+-- | Construct a 'Rope' out of a single 'ByteString' strand.
+ropeBS :: ByteString -> Rope
+ropeBS = rope . singleton . strand
```

Comment:
This is identical to the `unit` implementation of the `Reducer ByteString Rope` instance, and I'm not sure if it's worth creating another alias for this. In particular since I don't think the name is very telling.